tools/lib/tests/test_srreader.py

```diff
@@ -8,6 +8,9 @@
 NUM_SEGS = 17 # number of segments in the test route
 ALL_SEGS = list(np.arange(NUM_SEGS))
 TEST_ROUTE = "344c5c15b34f2d8a/2024-01-03--09-37-12"
+QLOG_SIZE = 11643
+RLOG_SIZE = 70577
+ALL_QLOG_SIZE = 191350
```

Comment:
these constants are really hard to update if you wanna change the route later. i'd just check stuff like len(qlog) < len(rlog)/10 or something